workflow/simulate/concat_txte_simulations.py

```diff
@@ -23,13 +23,31 @@
 from itertools import chain
 
 import pandas as pd
+import pyranges as pr
 
 # concat count matrices
 logger.info("Concatenating count matrices")
+genes_gtf = pr.read_gtf(snakemake.input.genes_gtf)
 te_counts = pd.read_csv(snakemake.input.te_counts[0], sep="\t", index_col=0)
 tx_counts = pd.read_csv(snakemake.input.tx_counts[0], sep="\t", index_col=0)
 pd.concat([te_counts, tx_counts], axis=0).to_csv(snakemake.output.counts, sep="\t")
 
+
+def summarize_genes(tx_counts):
+    # add a gene id column
+    genes_df = genes_gtf.df
+    genes_df.rename(columns={"transcript_id": "tx_id"}, inplace=True)
+    tx_counts = tx_counts.merge(
+        genes_df[["tx_id", "gene_id"]], on="tx_id"
+    ).drop_duplicates()
+    return tx_counts.groupby("gene_id").sum().drop(columns=["tx_id"])
+
+
+# TODO: check this
```

Comment:
@jjjoelle were you able to verify the output?

Reply:
Why this function in `concat_txte_simulations.py` and `simulate_te_counts.py`? Is it possible to remove that redundancy?

Reply:
I originally made the function in `simulate_te_counts.py` but then that file got deleted. I updated and deleted `simulate_te_counts.py`. 
 
 And the output should be correct